### DISCUSSION TITLE
Missing Zonedump_Interval:

### DISCUSSION THREAD

**Discussion by SmithAdamL** [ufodone/envisalink_new#54] [General]
Hello -

Love this updated version, and the ability to assign each device to its own area.  

One important item that is missing that I cant find in the new UI Config is the zonedump_interval.  While not important on a DSC board, this is important for Honeywell, as it is the update pulse for the EnvisaLink board to check the status of the zones from the HW board.  By default this is 30 seconds, but for my near real time requests, I have it set at every 5 seconds.  

Did I miss where this setup is in the new UI version?

Adam

**Comment by ufodone** [comment]
Hi @SmithAdamL, the zone timer dumps have been removed with this version and replaced with a different algorithm to handle zone updates on the Honeywell systems.  The zone timer dumps caused issues with zones flapping open/close due to them conflicting with other zone updates that the EVL generated.  I suspect you'll find the new algorithm just as responsive to updates as setting `zonedump_interval=5` on the old version.

There's some discussion [here](https://github.com/ufodone/envisalink_new/issues/9#issuecomment-1410972022) if you're interested in the details.
